builtin/mainmenu/dlg_create_world.lua

```diff
@@ -78,6 +78,17 @@ local mgv6_biomes = {
 	},
 }
 
+local mapgens_descriptions = {
+	v7 = fgettext("Default mapgen with large complex mountins and plains."),
+	valleys = fgettext("Large valleys with complex terrain and rivers."),
+	carpathian = fgettext("Realistic looking world with vast plains."),
+	v5 = fgettext("Old mapgen."),
+	flat = fgettext("World Flat terrain."),
+	fractal = fgettext("Wold with fractal structure."),
+	singlenode = fgettext("Empty world, use for lua defined mapgens."),
+	v6 = fgettext("Simple mapgen with few features, not recommended."),
```

Comment:
okay but [my game](https://github.com/sfan5/minetest_classic) only offers v6, so it doesn't make sense if the menu is saying that it's "not recommended". we should keep the description somewhat neutral.